resolwe_bio/processes/workflows/wgs_gvcf.py

```diff
@@ -61,33 +61,22 @@ class Input:
             default=False,
         )
 
-        class GatkOptions:
-            """Options."""
-
-            intervals = DataField(
-                "bed",
-                label="Intervals BED file",
-                description="Use intervals BED file to limit the analysis to "
-                "the specified parts of the genome.",
-                required=False,
-            )
-
-            contamination = IntegerField(
-                label="Contamination fraction",
-                default=0,
-                description="Fraction of contamination in sequencing "
-                "data (for all samples) to aggressively remove.",
-            )
-
         class Trimming:
             """Trimming parameters."""
 
+            enable_trimming = BooleanField(
+                label="Trim and quality filter input data",
+                description="Enable or disable adapter trimming and QC filtering procedure.",
+                default=False,
+            )
+
             adapters = DataField(
                 "seq:nucleotide",
                 label="Adapter sequences",
                 required=False,
                 description="Adapter sequences in FASTA format that will "
                 "be removed from the reads.",
+                disabled="!trimming_options.enable_trimming",
```

Comment:
I am not sure if `disabled`  parameter is working as expected. Could you please check if fields are disabled?

Reply:
The fields work as expected!